client/src/components/pages/RobotFile/RobotFile.jsx

```diff
@@ -35,14 +41,15 @@ const RobotFile = () => {
    * This function will retrieve the code from the editor, parse it to a ssot and write the
    * resulting ssot into the sessionStorage.
    */
-  const onSaveToCloud = async () => {
-    // eslint-disable-next-line no-unused-vars
-    const robotId = JSON.parse(sessionStorage.getItem('robotId'));
+  const onSaveToCloud = /* async */ () => {
```

Comment:
Please remove the comment here if it should stay an synchronous function or make it async again if needed

Reply:
is removed